packages/babel-preset-css-prop/README.md

```diff
@@ -139,6 +139,8 @@ Options for both `babel-plugin-emotion` and `@babel/plugin-transform-react-jsx`
 >
 > - [`@babel/plugin-transform-react-jsx`](https://babeljs.io/docs/en/next/babel-plugin-transform-react-jsx)
 
+You can opt into [the new JSX runtimes](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) by configuring this preset with `{ runtime: 'automatic' }`. Keep in mind that you have to use compatible React version (`>=16.14.0`) to use this option and a compatible version of `@emotion/core` (`>=10.1.0`).
```

Comment:
As I was reading through these, I noticed that it's ambiguous whether you're supposed to also include `@babel/preset-react`. (The only mention says "If you use `@babel/preset-react` [...]".) If I understand correctly, the story is a bit more complicated with the new runtime.

If you're using the classic runtime, then you "need" the preset because it includes the source/self transforms in dev and the pure annotations in prod. Whereas in the modern runtime, that functionality has all been absorbed into `@babel/plugin-transform-react-jsx{,-dev}`, so additionally applying the preset has no effect. Do you think it's worth adding a note to explain this?

Btw, I noticed that you had a related comment:

https://github.com/emotion-js/emotion/blob/71514b06fe172517168f98321499f05e74388de6/packages/babel-preset-css-prop/src/index.js#L32-L34

Not sure if you considered this but I think another advantage you'd get from switching to the preset would be that users _never_ need to apply the React preset themselves, regardless of the runtime. Maybe something to consider for Emotion 11 if you plan to deprecate Babel 6 support there.

Reply:
> Do you think it's worth adding a note to explain this?

Might be worth adding. Could you prepare a follow-up PR for this?

> Not sure if you considered this but I think another advantage you'd get from switching to the preset would be that users never need to apply the React preset themselves, regardless of the runtime.

Yeah, I'd like to do that - just need to verify how Babel merges presets etc. Ideally even if one has React preset configured but our comes later then it should just turn off the React preset. It works like this for plugins but I'm not sure if this happens when all presets are flattened into a plugins list or does it already happen at each step of resolution or smth.

> Maybe something to consider for Emotion 11 if you plan to deprecate Babel 6 support there.

Support for Babel 6 has been removed from Emotion 11.